spec/acceptance/nodesets/docker/ubuntu-12.04.yml

```diff
@@ -12,8 +12,6 @@ HOSTS:
     docker_image_commands:
       - 'apt-get install -y net-tools wget'
       - 'locale-gen en_US.UTF-8'
-      - 'ln -s /opt/puppetlabs/bin/facter /usr/bin/facter'
```

Comment:
@petems you added these 2 symlinks in the modulesync managed file with https://github.com/voxpupuli/puppet-collectd/commit/8dba7fc508696e72d8e513c7b62520960eff5a57 . is it really required?